includes/settings/settings-ppec.php

```diff
@@ -499,7 +499,7 @@ function showHideDefaultButtonSettings() {
 	'use_spb' => array(
 		'title'       => __( 'Smart Payment Buttons', 'woocommerce-gateway-paypal-express-checkout' ),
 		'type'        => 'checkbox',
-		'default'     => 'no',
+		'default'     => $this->get_option( 'button_size' ) ? 'no' : 'yes', // Hinge on whether settings have been initialized.
```

Comment:
Nit: being more specific in this comment would be helpful. E.g. `button_size` being set indicates an existing PPEC install.

Reply:
Done in https://github.com/woocommerce/woocommerce-gateway-paypal-express-checkout/pull/409/commits/383872cdbdb793ec8d7edca591aeb5a851385fba